web/docs/understand/language/expressions.md

```diff
@@ -185,6 +185,14 @@ Value predicates of type `subnet` expand more broadly. Given a subnet
 This makes it easier to search for IP addresses belonging to a specific subnet.
 :::
 
+### Extractor Predicates
+
+Similarly, predicates that test for the existance of a field or type can be
+written tersely as well. That is, every predicate `E != nil`, where `E` is a
+field or type extractor, can also be equivalently written as `E`. For example,
+if the field extractor `id.orig_h` is used as a predicate, it expands to
+`id.orig_h != nil`. The type extractor `:uint64` expands to `:uint64 = nil`.
```

Comment:
To be honest I find "Extractor Predicate" to be quite impenetrable.
It would make more sense to me if we explain this feature as a "syntactic sugar" or "short form" of type and field extractors respectively.

Reply:
It's an advanced concept anyway, so I think the "weight" of the terminology is not that important in this case.